Tests/Indicators/AlphaTests.cs

```diff
@@ -0,0 +1,75 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+using System;
+using static QuantConnect.Tests.Indicators.TestHelper;
+
+namespace QuantConnect.Tests.Indicators
+{
+    [TestFixture]
+    public class AlphaTests : CommonIndicatorTests<TradeBar>
+    {
+        protected override string TestFileName => "alpha_datatest.csv";
```

Comment:
Please use the test data provided in the original issue and adjust the test to suit